apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,15 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27156&commitId=0d5037766ef777520eaec65cddbce17732af2014&env=prod&clientType=FLUENTUI)


  <details><summary><b>ProgressBar converged</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressBar converged.Determinate - High Contrast.chromium.png|2888| Changed|
|ProgressBar converged.Indeterminate - RTL.chromium.png|831| Changed|
|ProgressBar converged.Indeterminate with thickness large.chromium.png|1662| Changed|
|ProgressBar converged.Indeterminate - High Contrast.chromium.png|3895| Changed|
|ProgressBar converged.Indeterminate.chromium.png|831| Changed|
|ProgressBar converged.Indeterminate - Dark Mode.chromium.png|3256| Changed|
|ProgressBar converged.Error.chromium.png|1553| Changed|
|ProgressBar converged.Determinate - Dark Mode.chromium.png|2731| Changed|
|ProgressBar converged.Determinate - RTL.chromium.png|1639| Changed|
|ProgressBar converged.Warning.chromium.png|1549| Changed|
|ProgressBar converged.Determinate with thickness large.chromium.png|3275| Changed|
|ProgressBar converged.Determinate.chromium.png|1639| Changed|
|ProgressBar converged.Success.chromium.png|1726| Changed|

</details>


  <details><summary><b>Skeleton converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton converged.Opaque Skeleton with circle - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Opaque Skeleton with circle - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Opaque Skeleton with circle - RTL.chromium.png|16| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - RTL.chromium.png|133| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - Dark Mode.chromium.png|23915| Changed|
|Skeleton converged.Opaque Skeleton with rectangle - High Contrast.chromium.png|28496| Changed|
|Skeleton converged.Opaque Skeleton with square - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Opaque Skeleton with square - RTL.chromium.png|23| Changed|
|Skeleton converged.Translucent Skeleton with  circle - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Opaque Skeleton with square - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Translucent Skeleton with  circle - RTL.chromium.png|25| Changed|
|Skeleton converged.Translucent Skeleton with  circle - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - High Contrast.chromium.png|28496| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - RTL.chromium.png|203| Changed|
|Skeleton converged.Translucent Skeleton with rectangle - Dark Mode.chromium.png|23915| Changed|
|Skeleton converged.Translucent Skeleton with square - Dark Mode.chromium.png|24985| Changed|
|Skeleton converged.Translucent Skeleton with square - High Contrast.chromium.png|29771| Changed|
|Skeleton converged.Translucent Skeleton with square - RTL.chromium.png|35| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>